projects/rero/ng-core/src/lib/record/detail/detail.component.html

```diff
@@ -14,27 +14,29 @@
  You should have received a copy of the GNU Affero General Public License
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<div class="alert alert-danger" *ngIf="error">{{ error | translate }}</div>
-<div class="float-right ml-4 mt-2 mb-4" *ngIf="record && adminMode.can">
-    <a href class="btn btn-sm btn-primary" routerLink="../../edit/{{ record.metadata.pid }}"
-        *ngIf="updateStatus && updateStatus.can">
-        <i class="fa fa-pencil"></i>
-        {{ 'Edit' | translate }}
-    </a>
-    <ng-container *ngIf="deleteStatus">
-        <button class="btn btn-sm btn-outline-danger ml-1" [title]="'Delete'|translate"
-            (click)="deleteRecord(record.metadata.pid)" *ngIf="deleteStatus.can; else deleteMessageLink">
-            <i class="fa fa-trash"></i>
-            {{ 'Delete' | translate }}
-        </button>
-        <ng-template #deleteMessageLink>
-            <button class="btn btn-sm btn-outline-danger ml-1 disabled" [title]="'Delete'|translate"
-                (click)="showDeleteMessage(deleteStatus.message)" *ngIf="deleteStatus.message">
-                <i class="fa fa-trash mr-1"></i>{{ 'Delete' | translate }}
-            </button>
-        </ng-template>
-    </ng-container>
+<div class="ui-content">
```

Comment:
I don't like `ui-content` because all the page is a user interface, not only the content. Please change to one of these: 
* `<div class="main-content">`
* `<div class="main-section">`
* `<div class="main">`